libmscore/read206.cpp

```diff
@@ -1361,10 +1375,22 @@ static bool readTextProperties206(XmlReader& e, TextBase* t, Element* be)
 
 static void readText206(XmlReader& e, TextBase* t, Element* be)
       {
+      // some default values differ from v2 to 3
+      t->setFrameRound(25);
```

Comment:
How did you define the default values? Where did you get them?

Reply:
I guess from the mtest 'failure' earlier ;-)

Reply:
@anatoly-os These default values aren't defined anywhere else in MS3, because they're the default values for frame style in MS2. So, since writing a frame with default values in MS2 leaves out any explicit specification for frame width, border radius etc., to prevent the look being changed we need to use the default values from MS2.

Reply:
How should they be defined? As a QVector of PIDs and MuseScore2 default values perhaps? Then they could be iterated over and set I guess.

Reply:
No, just separate constexpr variables with clear names.

Reply:
@anatoly-os done now